docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/field-label/field-label.scss

```diff
@@ -0,0 +1,74 @@
+//
+// Label component.
+//
+
+@use 'sass:map';
+@use 'sass:math';
+
+@mixin civictheme-field-label-size($size) {
+  // stylelint-disable-next-line scss/dollar-variable-pattern
+  $sizes: ('extra-large', 'large', 'regular', 'small', 'extra-small');
+
+  @if $size == 'extra-large' {
```

Comment:
please use 

```
  @if map.get($sizes, $size) == null {
    @error '"#{$size}" is not an available size.';
  }

```

Reply:
also, can we not use mixin and print this within a class 

having a mixin suggests that this can be re-used somewhere, which is not the case - Field label is a self-contained component and should only be used as a component